osimpipeline/study.py

```diff
@@ -44,31 +63,47 @@ class Trial(object):
 
     Examples
     --------
-
+    Passing no arguments creates a single trial object that begins and ends
+    with the times specified in the mocap data.
     ```
     new_trial = cond.add_trial(1)
 
     ```
 
+    Use the 'interval' dictionary argument to pick a specific start and/or end
+    time. The beginning or end time in the mocap file will be chosen if the
+    start or end time is left out of the argument, respectively.
     ```
-    new_trial = cond.add_trial(1, start_time=0.0, end_time=0.5)
+    interval = dict()
+    interval['start_time'] = 0.0
+    interval['end_time'] = 0.5
+    new_trial = cond.add_trial(1, interval=interval)
 
     ```
 
+    Use the 'gait_events' dictionary argument to provide the events
+    corresponding to individual gait cycles within a trial. A TrialTask will
+    be created for each individual cycle. Either the 'right_strikes' or
+    'left_strikes' entry is required to create gait cycles. If the desired gait
+    cycles in the trial are not consecutive, then the 'stride_times' entry is 
+    also required. The opposite foot heel strikes and toeoffs from both feet
+    can be included as optional dictionary entries.
```

Comment:
Also, I'm going to raise an issue about the optional toeoff dict entries. They're optional for the Trial object creation, but some of the postprocessing routines don't work without them.